DIPs/DIP1043.md

```diff
@@ -68,10 +68,10 @@ An implementation utilizing the proposed feature is only 8 lines:
 ```d
 struct ExclusiveRange(T)
 {
-	T from, to;
+   T from, to;
    invariant(from <= to);
    bool empty() => from == to;
-   auto popFront() => ++from;
+   void popFront() => ++from;
```

Comment:
This doesn't work with the current lowering of the DIP

Reply:
And I don't think it should - the current rule is very simple and consistent with the existing => functions.

Reply:
I see, changed back to auto.